### PR TITLE
Removed references to docker-machine in Swarm guide

### PR DESCRIPTION
**Pull request by nitisht**
## Description
Removed references to docker-machine in Swarm guide

## Motivation and Context
`docker-machine` command is not relevant for Swarm mode. It makes sense to remove all the references to `docker-machine` and just provide steps on using Swarm related commands. Also see https://github.com/minio/minio/issues/4477

## How Has This Been Tested?
Manually

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by harshavardhana — DISMISSED** [review]

File: docs/orchestration/docker-swarm/README.md [hunk 1]
```diff
@@ -10,19 +10,19 @@ As of [Docker Engine v1.13.0](https://blog.docker.com/2017/01/whats-new-in-docke
 * Docker engine v1.13.0 running on a cluster of [networked host machines](https://docs.docker.com/engine/swarm/swarm-tutorial/#/three-networked-host-machines).
 
 ## 2. Create a Swarm
-
-SSH into the machine supposed to serve as Swarm manager. If the machine is named `manager`, you can SSH by
+Logon to the machine supposed to work as Swarm manager machines, and initialize the Swarm by
```
Comment (krisis) [inline docs/orchestration/docker-swarm/README.md]: I think we can say this at a high-level as below.
```
Create a swarm on the manager node by,
<docker swarm command goes here>
```

File: docs/orchestration/docker-swarm/README.md [hunk 2]
```diff
@@ -10,19 +10,19 @@ As of [Docker Engine v1.13.0](https://blog.docker.com/2017/01/whats-new-in-docke
...
 
 ```shell
-docker swarm init --advertise-addr <MANAGER-IP>
+docker swarm join \
+  --token  SWMTKN-1-49nj1cmql0jkz5s954yi3oex3nedyz0fb0xx14ie39trti4wxv-8vxv8rssmk743ojnwacrr2e7c \
+  192.168.99.100:2377
 ```
-
-After the manager is up, [add worker nodes](https://docs.docker.com/engine/swarm/swarm-tutorial/add-nodes/) to the Swarm. Find detailed steps to create the Swarm on [Docker documentation site](https://docs.docker.com/engine/swarm/swarm-tutorial/create-swarm/).
+To add other nodes to the Swarm, run the above command to [add worker nodes](https://docs.docker.com/engine/swarm/swarm-tutorial/add-nodes/) to the Swarm. Find detailed steps to create the Swarm on [Docker documentation site](https://docs.docker.com/engine/swarm/swarm-tutorial/create-swarm/).
```
Comment (krisis) [inline docs/orchestration/docker-swarm/README.md]: We should refer to the container orchestration solution as the "Swarm" and the cluster of machines hosting docker containers amongst them as the "swarm". So, we should say, "To add other nodes to the swarm ...".


> To add other nodes to the Swarm, run the above command to [add worker nodes]

In this sentence, "to add worker nodes" is superfluous. We could say, "To add other nodes to the swarm, run the above [command](https://docs.docker.com/engine/swarm/swarm-tutorial/add-nodes/)."

**Review by krisis — CHANGES_REQUESTED** [review]

File: docs/orchestration/docker-swarm/README.md (hunk 2 above)
Comment (nitisht) [reply docs/orchestration/docker-swarm/README.md]: > We should refer to the container orchestration solution as the "Swarm" and the cluster of machines hosting docker containers amongst them as the "swarm".

Agreed. 

> In this sentence, "to add worker nodes" is superfluous. We could say, "To add other nodes to the swarm, run the above command."

Nodes themselves are of two types, and one can add new `master` or `worker` nodes based on their needs. But I see the repetition, I will rephrase it to make it simpler.

File: docs/orchestration/docker-swarm/README.md (hunk 1 above)
Comment (nitisht) [reply docs/orchestration/docker-swarm/README.md]: Done

**Comment by harshavardhana** [comment]
@krisis are there any other pending changes to this doc?

**Review by harshavardhana — DISMISSED** [review]

File: docs/orchestration/docker-swarm/README.md [hunk 3]
```diff
@@ -10,19 +10,20 @@ As of [Docker Engine v1.13.0](https://blog.docker.com/2017/01/whats-new-in-docke
...
 
 ```shell
-docker swarm init --advertise-addr <MANAGER-IP>
+docker swarm join \
+  --token  SWMTKN-1-49nj1cmql0jkz5s954yi3oex3nedyz0fb0xx14ie39trti4wxv-8vxv8rssmk743ojnwacrr2e7c \
+  192.168.99.100:2377
 ```
 
-After the manager is up, [add worker nodes](https://docs.docker.com/engine/swarm/swarm-tutorial/add-nodes/) to the Swarm. Find detailed steps to create the Swarm on [Docker documentation site](https://docs.docker.com/engine/swarm/swarm-tutorial/create-swarm/).
+You can now [add worker nodes](https://docs.docker.com/engine/swarm/swarm-tutorial/add-nodes/) to the swarm by running the above command. Find detailed steps to create the Swarm on [Docker documentation site](https://docs.docker.com/engine/swarm/swarm-tutorial/create-swarm/).
```
Comment (krisis) [inline docs/orchestration/docker-swarm/README.md]: s/create the Swarm/create the swarm

File: docs/orchestration/docker-swarm/README.md [hunk 4]
```diff
@@ -10,19 +10,20 @@ As of [Docker Engine v1.13.0](https://blog.docker.com/2017/01/whats-new-in-docke
...
-
-SSH into the machine supposed to serve as Swarm manager. If the machine is named `manager`, you can SSH by
+Create a swarm on the manager node by running
 
 ```shell
-docker-machine ssh manager
+docker swarm init --advertise-addr <MANAGER-IP>
 ```
-After logging in to the designated manager node, create the Swarm by
+Once the Swarm is initialized, you'll see the below response.
```
Comment (krisis) [inline docs/orchestration/docker-swarm/README.md]: s/Once the Swarm/Once the swarm

**Review by krisis — CHANGES_REQUESTED** [review]
Overall looks good to me except for couple of Swarm/swarm mix-ups.

File: docs/orchestration/docker-swarm/README.md (hunk 4 above)
Comment (nitisht) [reply docs/orchestration/docker-swarm/README.md]: done.

File: docs/orchestration/docker-swarm/README.md (hunk 3 above)
Comment (nitisht) [reply docs/orchestration/docker-swarm/README.md]: done

**Review by krisis — APPROVED** [review]

**Review by harshavardhana — APPROVED** [review]
